Makefile

```diff
@@ -342,9 +345,13 @@ endif
 # Current operator version
 VERSION ?= 0.37.0
 
+.PHONY: generate-operator-bundle
+generate-operator-bundle: LATEST_VERSION := $(shell curl -sL https://api.github.com/repos/Azure/azure-service-operator/releases/latest  | jq '.tag_name' --raw-output )
+generate-operator-bundle: LATEST_TAG := $(PUBLIC_REPO):$(LATEST_VERSION)
+generate-operator-bundle: LATEST_SHA := $(shell docker pull $(LATEST_TAG) > /dev/null && docker image inspect $(LATEST_TAG) | jq --raw-output '.[0].Id' )
 generate-operator-bundle: manifests
 	kustomize build config/manifests | operator-sdk generate bundle --version $(VERSION) --channels stable --default-channel stable --overwrite
 	# This is only needed until CRD conversion support is released in OpenShift 4.6.x/Operator Lifecycle Manager 0.16.x
 	scripts/add-openshift-cert-handling.sh
-	# Rather than modify config/rbac manifests, replace CSV's default serviceAccount with azure-service-operator
-	sed -i 's/serviceAccountName: default/serviceAccountName: azure-service-operator/g' bundle/manifests/azure-service-operator.clusterserviceversion.yaml
+	# Inject the SHA-based container reference into the bundle.
+	scripts/inject-container-reference.sh "$(PUBLIC_REPO)@$(LATEST_SHA)"
```

Comment:
What's the argument for SHA here vs TAG? Is it some OLM/OperatorHub requirement? If so maybe note that in the comment?

Reply:
I'm not sure whether it's required - I was maintaining it because that's what our existing one uses. 
Looking at bundles in the community-operators repo most of them don't use that, so I'll simplify it to match what we do for helm.